Hackney.Shared.HousingSearch.Tests/Hackney.Shared.HousingSearch.Tests.csproj

```diff
@@ -11,6 +11,11 @@
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
     <PackageReference Include="coverlet.collector" Version="1.2.0" />
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />    
```

Comment:
this is quite old, so perhaps worth upgrading to the latest version

Reply:
Done in [33368c0](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/33368c063eee1f017c5c5b8f8ba994c6dd837d15)